build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'com.github.ksoichiro.console.reporter' version '0.5.0'
+}
+
+apply plugin: 'java'
+
+group 'io.leonis'
+version '1.0-SNAPSHOT'
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+javadoc {
+    source = sourceSets.main.allJava
+    classpath = configurations.compile
+}
+
+dependencies {
+    // rx
+    compile 'io.projectreactor:reactor-core:3.0.5.RELEASE'
```

Comment:
`reactor-core` is at `3.1.1.RELEASE`. could you bump the version? haven't checked the other dependencies but those might be updated as well